cabal-testsuite/PackageTests/Check/ConfiguredPackage/Sanity/NoName/cabal.out

```diff
@@ -1,4 +1,8 @@
 # cabal check
-Warning: pkg.cabal:0:0: "name" field missing
 Error: [Cabal-7035]
-parse error
+Error encountered when parsing cabal file pkg.cabal:
+
+pkg.cabal:0:0: error:
```

Comment:
Where an expected field is missing, does it make sense to report the location when that field is not expected at location `:0:0`?